python/tk_multi_launchapp/base_launcher.py

```diff
@@ -87,6 +87,11 @@ def _register_launch_command(
         if command_name.endswith("..."):
             command_name = command_name[:-3]
 
+        if group and menu_name.startswith(group):
+            # for all other grouping situations, simply remove the group name
```

Comment:
So, if my group is Nuke, does that mean that I will need X 10.0v1, Assist 10.0v1, Studio 10.0v1 and so on?